src/transformers/modeling_longformer.py

```diff
@@ -754,9 +781,8 @@ def _init_weights(self, module):
 
 
 LONGFORMER_START_DOCSTRING = r"""
-
-    This model is a PyTorch `torch.nn.Module <https://pytorch.org/docs/stable/nn.html#torch.nn.Module>`__ sub-class.
-    Use it as a regular PyTorch Module and refer to the PyTorch documentation for all matter related to general
+    This model is a Tensorflow `torch.nn.Module <https://pytorch.org/docs/stable/nn.html#torch.nn.Module>`__ sub-class.
+    Use it as a regular Tensorflow Module and refer to the Tensorflow documentation for all matter related to general
```

Comment:
I understand your love for TensorFlow but I think this is still a PyTorch model!

```suggestion
    This model is a PyTorch `torch.nn.Module <https://pytorch.org/docs/stable/nn.html#torch.nn.Module>`__ sub-class.
    Use it as a regular PyTorch Module and refer to the PyTorch documentation for all matter related to general
```

Reply:
haha, my search and replace was in the wrong file -> thanks for spotting